packages/valory/connections/abci/connection.py

```diff
@@ -111,9 +111,7 @@ class _TendermintABCISerializer:
 
     @classmethod
     def encode_varint(cls, number: int) -> bytes:
-        """Encode a number in varint coding."""
-        # Shift to int64
-        number = number << 1
+        """Encode a number (uint64) in varint coding."""
```

Comment:
Messages are written to a byte stream using uin64 length delimiters instead of int64.